app/models/participant_profile/ecf.rb

```diff
@@ -180,14 +176,15 @@ def unfinished_with_billable_declaration?(cohort:)
   end
 
   def unfinished_with_no_billable_declaration?(cohort:)
+    return false if mentor? && FeatureFlag.active?(:closing_2022)
     return false if completed_training?
-    return false unless cohort == Cohort.active_registration_cohort
+    return false unless cohort&.start_year == Cohort::DESTINATION_START_YEAR_FROM_A_FROZEN_COHORT
     return false unless schedule&.cohort&.payments_frozen?
 
-    participant_declarations.none? { |declaration| declaration.billable? && declaration.completed? }
+    participant_declarations.none?(&:billable?)
```

Comment:
why are we removing the completed?

Reply:
Because it should have been removed last year. It is unnecessary